package.json

```
@@ -116,6 +116,11 @@
                     "default": false,
                     "description": "Use the same Bazel server for queries and builds. By default, vscode-bazel uses a separate server for queries so that they can be executed in parallel with builds. You can enable this setting if running multiple Bazel servers has a negative performance impact on your system, but you may experience degraded performance in Visual Studio Code for operations that require queries."
                 },
+                "bazel.outputBase": {
```

```suggestion
                "bazel.queryOutputBase": {
```

Just to make explicit the scope of this setting.